test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -132,6 +132,98 @@ Describe "Format-Table" -Tags "CI" {
                 $result | Should -Match "Jim\s+5678\s+False"
         }
 
+        It "Format-Table output should not be greater than console width" {
+            $consoleWidth = [console]::WindowWidth
+
+            if ($consoleWidth -eq 0) {
+                [console]::WindowWidth | Should -Be 0
+            }
+            else{
```

Comment:
I think it is more right to skip the test as follow:
```suggestion
        It "Format-Table output should not be greater than console width" -Pending: ([console]::WindowWidth -eq 0) {
```

Reply:
I went ahead and added the pending flag like you noted, and updated the PR description with the perf information I gathered previously.

Reply:
@nickkimbrough Thanks! I expected you add script perf win information :-)